README.md

```diff
@@ -2,6 +2,10 @@
 
 # Announcement - v4 was released 🎈
 
+
+## Motivation and usage examples
```

Comment:
Could you please make in level 1 header and move a bit down after the `Downloads` section? Currently it's under the announcements, while it's not logically related to v4.